packages/common/src/types/scheduler.ts

```diff
@@ -406,6 +406,7 @@ export type ValidateProjectPayload = {
     userUuid: string;
     organizationUuid: string | undefined;
     explores?: (Explore | ExploreError)[];
+    onlyTables?: boolean;
```

Comment:
Should we make this more flexible? An option where the user chooses what they want to validate. Possible options: tables, charts, dashboards.

Reply:
I have added a `--only` option which receives an array of `charts,dashboards,tables` + tests for this
Didn't use the `--include` option because there's already a `--exclude` one for models and it could be confusing 💪 

Also updated the video in the desc with the usage + validation check